pkg/api/metadata.go

```diff
@@ -22,11 +22,27 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"runtime/debug"
 
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
 	"github.com/getsentry/sentry-go"
 	"github.com/golang/glog"
 )
 
+func getCommitSha() (string, error) {
```

Comment:
Mind adding a comment explaining what this command does? (which commit sha it returns,  when empty string is returned)

Reply:
@valeriiashapoval can this method be documented as suggested in the previous comment?